src/app/search/search.component.html

```diff
@@ -135,24 +135,25 @@ <h5>{{ resultDataSource.data.length }} Result(s)</h5>
   <!-- Selected project list -->
   <div style="max-height:1000px;overflow:auto">
     <h5 class="float-left" style="margin-top:0px" *ngIf="getTotalItems() > 0">
-      {{ getTotalItems() | projectCount }} Selected
+      <!-- {{ getTotalItems() | projectCount }} Selected -->
     </h5>
 
     <div class="col-lg-12 text-right box-padding" *ngIf="getTotalItems() > 0">
-      <button mat-raised-button class="btn btn-success btn-mrgn btn-sm align-middle" (click)="addAll()">
+      <button mat-raised-button class="btn btn-success btn-mrgn btn-sm align-middle" *ngIf="!isFromOtherPage()" (click)="addAll()">
```

Comment:
@vishaljcj  Could you disable the button if no private collections exist as it throws an error. Also, uncertain behaviour. occasionally adds the project to public collections.

Here i created a collection with 3 versions and made them all public
![Screenshot from 2019-03-14 15-09-37](https://user-images.githubusercontent.com/11005120/54363680-ab3f2d00-466b-11e9-9032-ee429e3bb59c.png)

Here is a screenshot showing them all public
![Screenshot from 2019-03-14 15-11-14](https://user-images.githubusercontent.com/11005120/54363693-b1cda480-466b-11e9-88b6-bcaee2303095.png)

here i tried to add a new project to an existing collection and it shows me public collection
![Screenshot from 2019-03-14 15-11-35](https://user-images.githubusercontent.com/11005120/54363710-b7c38580-466b-11e9-8f51-3c4c44a270c0.png)

.

Reply:
this issue has been fixed now